### ISSUE TITLE
About Page - Make Section Links Clickable

### ISSUE THREAD

**Issue by suziebytes** [suziebytes/Resswell#6]
Make section link clickable  
On click -> displays section text 
On click for a different link -> old text disappears 

If text is too long to fit in text box, add scroll for text

**Comment by suziebytes** [comment]
- How many rows? 
- How many columns?

![services mockup](https://user-images.githubusercontent.com/32041993/52905020-a744fa00-31e8-11e9-8d63-8f9bfe3fb265.jpg)

**Comment by suziebytes** [comment]
Discussion with others on how best to approach this using grid! 
Looking to practically apply grid on this page and project with a focus on 2D (column/row) 

**QUESTIONS**
Is it best to make the white space a row or margin ? 
![image](https://user-images.githubusercontent.com/32041993/52905135-ab721700-31ea-11e9-9c61-2fb6f239a339.png)

It is apparent there are three columns: 
![image](https://user-images.githubusercontent.com/32041993/52905158-086dcd00-31eb-11e9-95bf-070422bce29e.png)

**Comment by suziebytes** [comment]
So I took a break and ended up watching this:
https://scrimba.com/g/gR8PTE


Great tutorial on CSS grid with short videos on different concepts of the grid. 
Explains why doing xyz is better than something else. Enjoy it! 
 
Better understanding of grids. Updated grid plan using `grid-areas`
There are 
11 rows  
9 different elements on the page designated by the letter
![image](https://user-images.githubusercontent.com/32041993/52908907-97063c80-3233-11e9-943c-08a7df0151ae.png)
